### PR TITLE
OADP-3652: fix CVE CVE-2024-24786

### PR DESCRIPTION
**Pull request by weshayutin**

### CODE REVIEW

**Review by sseago — DISMISSED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

File: go.sum
Comment (kaovilai) [inline go.sum:1]: Did you run `go mod tidy`?

/hold

Don't see any replacements in go.sum entry

Comment (kaovilai) [reply go.sum:1]: This pr is currently additive to go.sum

Comment (kaovilai) [reply go.sum:1]: I'm concerned about https://github.com/openshift/oadp-operator/pull/1367 as well

Comment (kaovilai) [reply go.sum:1]: If the changes were correctly tidied go sum should look like https://github.com/openshift/velero/pull/313

**Review by kaovilai — APPROVED** [review]
Ok now that's a tidy diff :)

/unhold
/lgtm

**Comment by kaovilai** [comment]
might wanna tidy the 1.3 branch too

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**